legend-pure-core/legend-pure-m3-core/src/test/java/org/finos/legend/pure/m3/tests/function/base/collection/AbstractTestMapCollection.java

```diff
@@ -15,10 +15,17 @@
 package org.finos.legend.pure.m3.tests.function.base.collection;
 
 import org.finos.legend.pure.m3.tests.AbstractPureTestWithCoreCompiled;
+import org.junit.After;
 import org.junit.Test;
 
 public abstract class AbstractTestMapCollection extends AbstractPureTestWithCoreCompiled
 {
+    @After
+    public void cleanRuntime()
+    {
+        AbstractPureTestWithCoreCompiled.runtime.delete("fromString.pure");
```

Comment:
We need to make sure to compile after deleting.
```suggestion
        runtime.delete("fromString.pure");
        runtime.compile();
```